source/data-formats/bson.txt

```diff
@@ -0,0 +1,89 @@
+.. _kotlin-sync-bson:
+
+==========================
+Document Data Format: BSON
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about the BSON data format, how MongoDB
+uses it, and how to install the BSON library independently of the
+{+driver-short+}.
+
+BSON Data Format
+----------------
+
+**BSON**, or Binary JSON, is the data format that MongoDB uses to organize
+and store data. This data format includes all JSON data structure types and
+adds support for types including dates, differently-sized integers, ObjectIds, and
+binary data. For a complete list of supported types, see the
+:manual:`BSON Types </reference/bson-types>` server manual page.
+
+The binary format is not human-readable, but you can use the
+:ref:`BSON library <install-bson-library>` to convert it to the human-readable JSON
+representation. You can read more about the relationship between these
+formats in the :website:`JSON and BSON </json-and-bson>` guide on the MongoDB website.
+
+MongoDB and BSON
+----------------
+
+You can use BSON data with your {+driver-short+} application by using one of the
+following object types that implements the `BSON interface <{+java-api+}/apidocs/bson/org/bson/conversions/Bson.html>`__:
+
+- `Document <{+java-api+}/apidocs/bson/org/bson/Document.html>`__ (BSON library package)
+- `BsonDocument <{+java-api+}/apidocs/bson/org/bson/BsonDocument.html>`__ (BSON library package)
+- `RawBsonDocument <{+java-api+}/apidocs/bson/org/bson/RawBsonDocument.html>`__ (BSON library package)
+- `JsonObject <{+java-api+}/apidocs/bson/org/bson/json/JsonObject.html>`__ (BSON library package)
+
+.. _install-bson-library:
+
+Install the BSON Library
+------------------------
+
+These instructions detail how to add the BSON library as a dependency to
+your project.
+
+.. note::
+
+   If you have already added the {+driver-short+} as a dependency to your
+   project, then you can skip this step. This is because the BSON library is already
+   included as a required dependency of the driver.
+
+.. TODO: For instructions on how to add the
+.. MongoDB Kotlin driver as a dependency to your project, see the
+.. :ref:`driver installation <kotlin-sync-download-install>` section of our Get Started
+.. guide.
+
+We recommend that you use the `Maven <https://maven.apache.org/>`__ or
+`Gradle <https://gradle.org/>`__ build automation tool to manage your {+language+}
+project's dependencies. Select from the following tabs to see the dependency declaration
```

Comment:
Q: I'm not sure if you need the sentence "Select from the following tabs...", as it seems to be pretty clear which tab does what/how users would get to a certain tab. 

S: Remove the sentence

Reply:
I think there should still be some sort of introduction to the tabbed instructions so I'm opposed to removing the sentence outright, but I can tweak it to be a little more natural-sounding.